site/en/userGuide/search/hybridsearch.md

```diff
@@ -73,7 +77,11 @@ const params = {
 await milvusClient.collectionManager.createCollection(params);
 ```
 
-3. Insert data into the collection. See [Manage Data](manage_data.md) for more instruction.
+```cli
+create collection -c test_book_search -f book_intro:FLOAT_VECTOR:2 -f book_id:INT64 -f word_count::INT64 -p book_id
```

Comment:
`create collection -c test_book_search -f book_intro:FLOAT_VECTOR:2 -f book_id:INT64 book_id -f word_count:INT64 word_count -p book_id`

Reply:
`<fieldName>:<dataType>:<dimOfVector/desc>`. We need description for scalar fields. Confirm with @czhen-zilliz .